internal/graph/check_fast_path.go

```diff
@@ -928,7 +928,7 @@ func (c *LocalChecker) recursiveFastPath(ctx context.Context, req *ResolveCheckR
 
 	userIter, err := checkutil.IteratorReadStartingFromUser(ctx, typesys, ds, req,
 		tuple.ToObjectRelationString(tuple.GetType(req.GetTupleKey().GetObject()), req.GetTupleKey().GetRelation()),
-		nil)
+		nil, false)
```

Comment:
A bit concern that in some cases the iterator read starting from user is not sorted - would it lead to error if the caller relies on it in some path.

Reply:
This is funny. @elbuo8 told me that putting `true` here was unnecessary. https://github.com/openfga/openfga/pull/2150#discussion_r1892402696

You both have a point. I guess I'll add a godoc explaining why this is false

Reply:
Updated existing tests.